spk/pyload/src/app/config

```diff
@@ -0,0 +1,12 @@
+{
+    ".url": {
+        "com.synocommunity.packages.pyload": {
+            "title": "pyLoad",
+            "desc": "pyLoad",
+            "icon": "images/pyload-{0}.png",
+            "type": "url",
+            "url": "3rdparty/pyload/pyload.cgi",
```

Comment:
Well, how about removing the complication :) If nothing else, predetermining a port removes ambiguity for less experienced users, and helps us with package support, because everyone basically starts off with the same configuration.

So, I'd propose to set the port to a fixed (free) port. Check the wiki pages to be sure which ports are available. You can use the directions at the top to determine which port to use, probably somewhere in the 81xx range or so. Then, use that port for `ADMIN_PORT`, the service configure file and the config/shortcut.